### PR TITLE
Like counter

### PR DESCRIPTION
**Pull request by doraby**
Add like button and list of the likers
Add static favorite button
Use Button component from the and.design
Connect ant.design css to theme

### CODE REVIEW

File: src/Components/Answer/Answer.js [hunk 1]
```diff
@@ -4,23 +4,40 @@ import PropTypes from 'prop-types';
...
+        }
+    }
+
+    handleLike = answer => {
+        let answerClone = this.state.answer;
+        answerClone.liked = !answer.liked;
+        const index = answer.likerList.indexOf(this.props.username);
+        answer.liked ? answer.likerList.push(this.props.username) : answer.likerList.splice(index,1);
+        this.setState({ answer });
+        console.log(this.state.answer.likerList);
```
Comment (AlexDovgalenok) [inline src/Components/Answer/Answer.js]: Please, remove debuging code

File: src/Components/Answer/Answer.js [hunk 2]
```diff
@@ -4,23 +4,40 @@ import PropTypes from 'prop-types';
...
+    };
+
     render() {
         const handleClick = this.props.handleClick;
-        const id = this.props.key;
+        const key = this.props.key;
+        const arrayQuestion = this.props.questionId;
+        console.log(arrayQuestion);
```
Comment (AlexDovgalenok) [inline src/Components/Answer/Answer.js]: The same

File: src/Components/Answer/Answer.js [hunk 3]
```diff
@@ -4,23 +4,40 @@ import PropTypes from 'prop-types';
...
+        super(props);
+        this.state = {
+            answer: this.props.answer,
+        }
+    }
+
+    handleLike = answer => {
+        let answerClone = this.state.answer;
+        answerClone.liked = !answer.liked;
+        const index = answer.likerList.indexOf(this.props.username);
```
Comment (AlexDovgalenok) [inline src/Components/Answer/Answer.js]: Will be better iif you declare all variables on one place at the top of function

File: src/Components/Answer/Answer.js (hunk 1 above)
Comment (doraby) [reply src/Components/Answer/Answer.js]: As you wish my lord

**Pull request merged**